.github/config/android-arm.json

```diff
@@ -0,0 +1,14 @@
+{
+  "docker_service": "build-android",
+  "platforms": [
+    "android-arm"
+  ],
+  "includes": [
+    {
+      "platform":"android-arm",
+      "target_platform":"android-arm",
+      "target_cpu":"target_cpu=\\\"arm\\\"",
```

Comment:
I think instead of relying on GN defaults we should add `target_cpu` and `target_os` to every configuration. That'll also mean we can get just have the syntax `"target_cpu":"arm"`

Reply:
Since some of the platform variants do not have target_cpu and/or target_os defined, can you help me get the values for those?  For instance:

* Evergreen and Raspi variants do not have target_os.
* Linux variants do not have neither.